csharp/private/frameworks/net60.BUILD

```diff
@@ -0,0 +1,1442 @@
+# This file was generated by build\src\D2L.Build.BazelGenerator\Commands\GenerateFrameworkBuildFile.cs
+# Don't edit it by hand! You should regenerate this if the .NET package gets updated.
+
+load("@d2l_rules_csharp//csharp:defs.bzl", "csharp_library_set", "import_library")
+
+csharp_library_set(
+    name = "StandardReferences",
```

Comment:
I forget if this list is just always everything? Generated the list from other target names.

Reply:
Compared to the list passed to csc for an empty project. Same items, different ordering.

```diff
diff --git a/csharp/private/frameworks/net60.BUILD b/csharp/private/frameworks/net60.BUILD
index 76019a6..171b0ee 100644
--- a/csharp/private/frameworks/net60.BUILD
+++ b/csharp/private/frameworks/net60.BUILD
@@ -8,32 +8,31 @@ csharp_library_set(
     target_framework = "net6.0",
     deps = [
         "//:Microsoft.CSharp",
-        "//:Microsoft.VisualBasic",
         "//:Microsoft.VisualBasic.Core",
+        "//:Microsoft.VisualBasic",
         "//:Microsoft.Win32.Primitives",
         "//:Microsoft.Win32.Registry",
         "//:mscorlib",
         "//:netstandard",
-        "//:System",
         "//:System.AppContext",
         "//:System.Buffers",
-        "//:System.Collections",
         "//:System.Collections.Concurrent",
+        "//:System.Collections",
         "//:System.Collections.Immutable",
         "//:System.Collections.NonGeneric",
         "//:System.Collections.Specialized",
-        "//:System.ComponentModel",
         "//:System.ComponentModel.Annotations",
         "//:System.ComponentModel.DataAnnotations",
+        "//:System.ComponentModel",
         "//:System.ComponentModel.EventBasedAsync",
         "//:System.ComponentModel.Primitives",
         "//:System.ComponentModel.TypeConverter",
         "//:System.Configuration",
         "//:System.Console",
         "//:System.Core",
-        "//:System.Data",
         "//:System.Data.Common",
         "//:System.Data.DataSetExtensions",
+        "//:System.Data",
         "//:System.Diagnostics.Contracts",
         "//:System.Diagnostics.Debug",
         "//:System.Diagnostics.DiagnosticSource",
@@ -44,27 +43,28 @@ csharp_library_set(
         "//:System.Diagnostics.Tools",
         "//:System.Diagnostics.TraceSource",
         "//:System.Diagnostics.Tracing",
+        "//:System",
         "//:System.Drawing",
         "//:System.Drawing.Primitives",
         "//:System.Dynamic.Runtime",
         "//:System.Formats.Asn1",
-        "//:System.Globalization",
         "//:System.Globalization.Calendars",
+        "//:System.Globalization",
         "//:System.Globalization.Extensions",
-        "//:System.IO",
-        "//:System.IO.Compression",
         "//:System.IO.Compression.Brotli",
+        "//:System.IO.Compression",
         "//:System.IO.Compression.FileSystem",
         "//:System.IO.Compression.ZipFile",
-        "//:System.IO.FileSystem",
+        "//:System.IO",
         "//:System.IO.FileSystem.AccessControl",
+        "//:System.IO.FileSystem",
         "//:System.IO.FileSystem.DriveInfo",
         "//:System.IO.FileSystem.Primitives",
         "//:System.IO.FileSystem.Watcher",
         "//:System.IO.IsolatedStorage",
         "//:System.IO.MemoryMappedFiles",
-        "//:System.IO.Pipes",
         "//:System.IO.Pipes.AccessControl",
+        "//:System.IO.Pipes",
         "//:System.IO.UnmanagedMemoryStream",
         "//:System.Linq",
         "//:System.Linq.Expressions",
@@ -87,13 +87,13 @@ csharp_library_set(
         "//:System.Net.WebClient",
         "//:System.Net.WebHeaderCollection",
         "//:System.Net.WebProxy",
-        "//:System.Net.WebSockets",
         "//:System.Net.WebSockets.Client",
+        "//:System.Net.WebSockets",
         "//:System.Numerics",
         "//:System.Numerics.Vectors",
         "//:System.ObjectModel",
-        "//:System.Reflection",
         "//:System.Reflection.DispatchProxy",
+        "//:System.Reflection",
         "//:System.Reflection.Emit",
         "//:System.Reflection.Emit.ILGeneration",
         "//:System.Reflection.Emit.Lightweight",
@@ -104,9 +104,9 @@ csharp_library_set(
         "//:System.Resources.Reader",
         "//:System.Resources.ResourceManager",
         "//:System.Resources.Writer",
-        "//:System.Runtime",
         "//:System.Runtime.CompilerServices.Unsafe",
         "//:System.Runtime.CompilerServices.VisualC",
+        "//:System.Runtime",
         "//:System.Runtime.Extensions",
         "//:System.Runtime.Handles",
         "//:System.Runtime.InteropServices",
@@ -119,7 +119,6 @@ csharp_library_set(
         "//:System.Runtime.Serialization.Json",
         "//:System.Runtime.Serialization.Primitives",
         "//:System.Runtime.Serialization.Xml",
-        "//:System.Security",
         "//:System.Security.AccessControl",
         "//:System.Security.Claims",
         "//:System.Security.Cryptography.Algorithms",
@@ -129,22 +128,23 @@ csharp_library_set(
         "//:System.Security.Cryptography.OpenSsl",
         "//:System.Security.Cryptography.Primitives",
         "//:System.Security.Cryptography.X509Certificates",
+        "//:System.Security",
         "//:System.Security.Principal",
         "//:System.Security.Principal.Windows",
         "//:System.Security.SecureString",
         "//:System.ServiceModel.Web",
         "//:System.ServiceProcess",
-        "//:System.Text.Encoding",
         "//:System.Text.Encoding.CodePages",
+        "//:System.Text.Encoding",
         "//:System.Text.Encoding.Extensions",
         "//:System.Text.Encodings.Web",
         "//:System.Text.Json",
         "//:System.Text.RegularExpressions",
-        "//:System.Threading",
         "//:System.Threading.Channels",
+        "//:System.Threading",
         "//:System.Threading.Overlapped",
-        "//:System.Threading.Tasks",
         "//:System.Threading.Tasks.Dataflow",
+        "//:System.Threading.Tasks",
         "//:System.Threading.Tasks.Extensions",
         "//:System.Threading.Tasks.Parallel",
         "//:System.Threading.Thread",
```